polkadot/node/core/backing/src/lib.rs

```diff
@@ -1728,6 +1752,12 @@ async fn kick_off_validation_work<Context>(
 	background_validation_tx: &mpsc::Sender<(Hash, ValidatedCandidateCommand)>,
 	attesting: AttestingData,
 ) -> Result<(), Error> {
+	// Do nothing if the local validator is disabled
+	if rp_state.table_context.local_validator_is_disabled() == Some(true) {
```

Comment:
if it's `None` that means we are not a validator? should we also exit in that case?

Reply:
By exit you mean not to kick off validation work, right? Not to stop the whole node?

Reply:
yep

Reply:
Yes. This makes sense.